_posts/0000-01-02-skylarartfall.md/Welcome to our second slide

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
+  My dreams is "I want crate my own AI Assistant to my PC, Laptop and Cellphone